src/components/App/App.js

```diff
@@ -1,36 +1,45 @@
 import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
 import './App.css';
 import Nav from '../Nav/Nav';
 import Cities from '../Cities/Cities';
-import SearchBar from '../SearchBar/SearchBar'
-import { cityData } from '../../cityData';
-
+import SearchBar from '../SearchBar/SearchBar';
+import CityDetails from '../CityDetails/CityDetails';
+import { getAllCities } from '../../apiCalls';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      cities: cityData.cities,
+      cities: [],
+      error: '',
     }
   }
 
-  // getAllCities = () => {
-  //   fetch('path')
-  //   .then(response => response.json())
-  //   .then(data => this.setState({ cities: data }))
-  //   .catch(error => console.log(error))
-  // };
-
+  componentDidMount = () => {
+    getAllCities() 
+      .then(data => this.setState({ cities: data }))
+      .catch(error => console.log(error))
```

Comment:
We can set the state of the errors to show a warning if there's a problem with our data loading.  Let's throw some conditional rendering down at line 32 like this: `{!error && <Cities />}`. 😎